.github/workflows/cross-compiles.yml

```diff
@@ -97,6 +97,12 @@ jobs:
             libs: libc6-dev-sh4-cross,
             target: no-async linux-latomic,
             tests: -test_includes -test_store -test_x509_store
+          }, {
+            arch: sparc64-linux-gnu,
+            libs: libc6-dev-sparc64-cross,
+            # work around a qemu bug
+            target: -static -fno-tree-vectorize linux64-sparcv9,
+            tests: "\" OPENSSL_sparcv9cap=\"0x120:0"
```

Comment:
I can't make sense of this.  Why isn't the following enough?

``` yaml
            tests: "OPENSSL_sparcv9cap=0x120:0"
```

Reply:
Because it is a hack. :grin: Your proposal would call `make test TESTS="OPENSSL_sparcv9cap=0x120:0"`

Reply:
or at the least a comment explaining it :)